src/Serilog.Extras.Timing/Extras/Timing/HealthCheckResult.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2014 Serilog Contributors
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using Serilog.Events;
+
+namespace Serilog.Extras.Timing
+{
+
+	/// <summary>
+	/// The result data of a health check.
+	/// </summary>
+	public class HealthCheckResult{
```

Comment:
Nitpick, but to avoid a bunch of churn/inconsistency when folks edit this using VS's conventions, can we place opening `{` on its own line? There's a _ReSharper_ settings file with the solution, if you like I can run an automatic reformat pre-merge?